packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1070,6 +1070,12 @@ oie.custom_app.push.sent = Push notification sent
 oie.custom_app.push.resend = Resend push notification
 # {0} refers to the display name of the authenticator
 oie.custom_app.push.warning = Haven't received a push notification yet? Try opening {0} on your phone.
+oie.authenticator.custom_app.method.push.verify.enable.biometrics = Your response was received, but your organization requires biometrics—like a fingerprint or facial scan—for access. Make sure your device meets the following requirements, then try again:
+oie.authenticator.custom_app.method.push.verify.enable.biometrics.title = Enable biometrics in {0}
+oie.authenticator.custom_app.method.push.verify.enable.biometrics.description = Your response was received, but your organization requires biometrics—like a fingerprint or facial scan—for access. Make sure your device meets the following requirements, then try again:
+oie.authenticator.custom_app.method.push.verify.enable.biometrics.point1 = Your device supports biometrics
+oie.authenticator.custom_app.method.push.verify.enable.biometrics.point2 = {0} is up-to-date
```

Comment:
please give a comment on what the parameter is referring to, this is needed when translating.